CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+### Added
```

Comment:
Did the semantics of the build order change in this PR? I recall you talking about this

Reply:
No